rmf_robot_sim_common/src/slotcar_common.cpp

```diff
@@ -362,6 +369,9 @@ void SlotcarCommon::handle_ackermann_path_request(
         Eigen::Vector2d(tangent1.x(), tangent1.y()),
         Eigen::Vector2d(0, 0),
         true);
+      if (locations[i-1].obey_approach_speed_limit)
+        turn_traj.approach_speed_limit = locations[i-1].approach_speed_limit;
```

Comment:
`approach_speed_limit` should be check that it is larger than 0, as it looks like it is not checked further down the code path, https://github.com/open-rmf/rmf_simulation/blob/feature/speed_limits/rmf_robot_sim_common/src/utils.cpp#L32